### ISSUE TITLE
horizontal parent() fail

### ISSUE THREAD

**Issue by olets** [twigjs/twig.js#382]
This might be new, or might be the simplest case of #352:

Given trunk.html.twig*

```
{% use branch.html.twig %}
{% block horizontal parent() %}
```

and branch.html.twig

```
{% block horizontal %}
    content
{% endblock %}
```

trunk.html.twig will output `{{|PARENT|}}` (caps and vertical bars sic)

*same with `{% block horizontal %}{{parent()}}{% endblock %}`

**Comment by olets** [comment]
Saw #257, which fully encompasses this, after posting. Leaving this open for now because of possible relation to #352 but feel free to close it if it's redundant

**Comment by willrowe** [comment]
Fixed by #579.
